flask_boggle/app.py

```diff
@@ -13,4 +14,16 @@ def start_page():
 @app.route('/start')
 def boggle_page():
     make_board = boggle_game.make_board()
-    return render_template('table.html',board = make_board)
+    session['board'] = make_board
+    high_score = session.get('high_score',0)
+    nplays = session.get('nplayes',0)
```

Comment:
Do you have a typo here? `nplays` vs `nplayes`

Reply:
Yeah, thank you to give the attention on that